src/project.coffee

```diff
@@ -96,11 +110,45 @@ class Project extends Model
 
   # Public: Get an {Array} of {GitRepository}s associated with the project's
   # directories.
+  #
+  # This method will be removed in 2.0 because it does synchronous I/O.
+  # Prefer the following, which evaluates to a Promise that resolves to an
+  # {Array} of {Repository} objects:
+  # ```
+  # project.getDirectories().map(
+  #     project.repositoryForDirectory.bind(project))
+  # ```
   getRepositories: -> _.compact([@repo])
   getRepo: ->
     Grim.deprecate("Use ::getRepositories instead")
     @repo
 
+  # Public: Returns a {Promise} that resolves with either:
```

Comment:
@bolinfest You can find the documentation on the AtomDoc format here:

https://github.com/atom/atomdoc

I'm probably going to be enhancing the documentation of it over the next couple weeks.